src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -5042,10 +5047,23 @@ namespace WaterThermalTanks {
 		Qnet = 0.0;
 		Qfuel = 0.0;
 
-		// Calculate steady-state heat rates
+		// Calculate the heating rate from the heat pump.
+		if ( Tank.HeatPumpNum > 0 ) {
+            HeatPumpWaterHeaterData const &HeatPump = HPWaterHeater(Tank.HeatPumpNum);
+			DataLoopNode::NodeData const &HPWHCondWaterInletNode = DataLoopNode::Node(HeatPump.CondWaterInletNode);
+			DataLoopNode::NodeData const &HPWHCondWaterOutletNode = DataLoopNode::Node(HeatPump.CondWaterOutletNode);
+			HPWHCondenserDeltaT = HPWHCondWaterOutletNode.Temp - HPWHCondWaterInletNode.Temp;
+		} else {
+			HPWHCondenserDeltaT = 0.0;
+		}
+        assert( HPWHCondenserDeltaT >= 0 );
```

Comment:
And then expect death on that newly modified failing integration test file!!  Just like [here](https://github.com/NREL/EnergyPlus/blob/develop/testfiles/CMakeLists.txt#L177).  Code coverage, @mbadams5 !